doc/Support/FAQ.md

```diff
@@ -544,6 +545,12 @@ define custom types, we suggest using [Device
 Groups](/Extensions/Device-Groups/). They will be listed in the
 menu similarly to device types.
 
+## <a name="faq34"> Editing large dynamic device groups gives error messages</a>
```

Comment:
It's not related to dynamic, it applies to static as well.
In fact, for dynamic the device array that triggers this it not even needed.

So need to rewrite this to be generic about "Large device groups"